### DISCUSSION TITLE
RfC - foster 3rd party support offerings for enterprise teams

### DISCUSSION THREAD

**Discussion by macolo** [django-cms/django-cms#7883] [General]
## Why
- provide additional security to potential corporate users and agencies that would like to have a robust 2nd level support layer as a prerequisite to adopt django CMS.

## What
- an offering for 2nd level and expert support for implementing / maintaining agencies and corporate end users of django CMS
- maybe a fixed price offering, i.e. 1000 USD / month, 5h support and coaching included.
- resources / time is provided by a trusted partner, billing is done directly by the partner.

## How
- nominate and endorse trusted partners (individuals / agencies) to act as service level providers
- formulate a standard service level agreement template
- publish a django-cms.org page detailing the offer

**Comment by jrief** [comment]
@macolo do you have any real world examples for this business model? My experience is, that stakeholders build a relationship with the agency they trust. For them, an association such as the dCA often is too abstract and too far away.

**Reply by macolo** [reply]
hi @jrief actually I don't. :) what you write makes a lot of sense to me. Let's explore in that direction.
